control/tests/xferfcn_test.py

```diff
@@ -404,34 +404,6 @@ def test_slice(self):
         assert (sys1.inputs, sys1.outputs) == (2, 1)
         assert sys1.dt == 0.5
 
-    @pytest.mark.parametrize("omega, resp",
-                             [(1, np.array([[-0.5 - 0.5j]])),
-                              (32, np.array([[0.002819593 - 0.03062847j]]))])
-    @pytest.mark.parametrize("dt", [None, 0, 1e-3])
-    def test_evalfr_siso(self, dt, omega, resp):
-        """Evaluate the frequency response at single frequencies"""
-        sys = TransferFunction([1., 3., 5], [1., 6., 2., -1])
-
-        if dt:
-            sys = sample_system(sys, dt)
-            s = np.exp(omega * 1j * dt)
-        else:
-            s = omega * 1j
-
-        # Correct versions of the call
-        np.testing.assert_allclose(evalfr(sys, s), resp, atol=1e-3)
-        np.testing.assert_allclose(sys(s), resp, atol=1e-3)
-        # Deprecated version of the call (should generate warning)
-        with pytest.deprecated_call():
-            np.testing.assert_allclose(sys.evalfr(omega), resp, atol=1e-3)
-
-        # call above nyquist frequency
-        if dt:
-            with pytest.warns(UserWarning):
-                np.testing.assert_allclose(sys._evalfr(omega + 2 * np.pi / dt),
-                                           resp,
-                                           atol=1e-3)
```

Comment:
keep an equivalent of this somewhere? Maybe it is already there?

Reply:
is it not moved down jst below to line 429 below as test_call_siso?

Reply:
see above